pkg/apis/broker/v1beta1/test_helper.go

```diff
@@ -67,31 +67,31 @@ func (t testHelper) AvailableEndpoints() *corev1.Endpoints {
 	}
 }
 
-func (t testHelper) ReadyDependencyStatus() *duckv1.KResource {
-	kr := &duckv1.KResource{}
+func (t testHelper) ReadyDependencyStatus() *duckv1.Source {
+	kr := &duckv1.Source{}
```

Comment:
nit: we can replace all `kr` in this file with like `src`.

Reply:
Makes sense, have addressed that.